### ISSUE TITLE
enable HISTORY=ON by default?

### ISSUE THREAD

**Issue by boeckmann** [SvarDOS/bugz#141]
A user (Willi Spiegl) asked if HISTORY=ON could be enabled by default. Not sure if it is the best thing to do, as it costs RAM. But a commented out entry in the default config.sys would be nice, with a description what it does. Then the user can enable it easily.

**Comment by mateuszviste** [comment]
I did not know myself about this EDR feature. It's awesome! :) Until now I was using [DOSKEY](http://paulhoule.com/doskey) for that, but the kernel's ````HISTORY=ON```` is less RAM hungry so I will use this from now on. Apparently it uses 576 bytes of RAM and stores the last 15 commands or so.

Should it be enabled by default? I'm not sure, but I am really tempted to make it a default . 576 bytes is not much - even on a 256K machine it's just 0.2% of the total RAM. This will definitely not block the installation process, and once the system is done people that are really low on RAM can easily comment it out. I bet the majority of users wants a shell history to be available one way or another.

**Comment by mateuszviste** [comment]
also, would be nice to have some page in the HELP about the available (EDR) kernel configurations..

**Comment by boeckmann** [comment]
For me its literally the first thing I add to config.sys. So I am ok with including it :)

The kernel configuration really would benefit from some documention. I bet not many will be able to build config.sys boot menues for SvarDOS EDR :)

**Comment by mateuszviste** [comment]
HISTORY=ON is enabled by default in bleeding edge and the generated CONFIG.SYS has a few words about what it is about. The default is limited to 256 bytes of history, I hope it's fine for @fritzmueller :-)

I have also initiated some basic kernel documentation:
http://svardos.org/phpamb.php?fname=help/help-en&f=kerncfg.ama

Ultimately each setting should have a line or two of meaningful description, but we have to start somewhere.
